packages/core/src/external/athenahealth/index.ts

```diff
@@ -257,11 +271,23 @@ class AthenaHealthApi {
           })
           .catch(processAsyncError("Error saving to s3 @ AthenaHealth - getPatientViaSearch"));
       }
-      const searchSet = patientSearchResourceSchema.parse(response.data);
-      if (searchSet.entry.length > 1) {
-        throw new NotFoundError("More than one AthenaHealth patient found");
+      const searchSet = patientSearchResourceSchema.safeParse(response.data);
+      if (!searchSet.success) {
+        capture.message(`Patient search set could not be parsed @ AthenaHealth`, {
+          extra: {
+            url: patientSearchUrl,
+            cxId,
+            practiceId: this.practiceId,
+            patientId,
+            context: "athenahealth.get-patient",
+          },
+          level: "info",
+        });
+        return undefined;
       }
-      return searchSet.entry[0]?.resource;
+      const entry = searchSet.data.entry;
+      if (entry.length > 1) throw new NotFoundError("More than one AthenaHealth patient found");
```

Comment:
Not found for more than one is odd, no? Should we throw a 500, like MetriportError?

Even worse, we don't throw NotFoundError when there's no patient (below).

Reply:
Can throw it from the core class as well, just a preference thing in my mind (having the core methods return undefined isn't "wrong" per se in my mind).

Reply:
no worries w/ upstream checking the return value and throwing there, the comment was just related to the fact we didn't throw NF on no patient but we did if there were more than one. All good then.